### PR TITLE
[hipercool] fix language typo + add proper logo

### PR DESCRIPTION
**Pull request by XenonR**

### CODE REVIEW

**Review by RenanMenck — COMMENTED** [review]
if possible, add the website logo as well: https://hiper.cool/favicon.png?v=3

File: src/web/mjs/connectors/HiperCool.mjs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            try {
+                request = new Request(new URL('/home/' + page, this.url), this.requestOptions);
+                data = await this.fetchDOM(request, '.news-list > li');
+                page++;
+
+                for (let manga of data) {
+                    mangas.push(
+                        {
+                            id: manga.querySelector('a.news-thumb').pathname,
+                            title: manga.querySelector('div.title').innerText.trim().replace(/\s+\d{2}(?: Final)?$/, '')
```
Comment (27136773) [inline src/web/mjs/connectors/HiperCool.mjs]: May place `trim` at the end in case spaces emerge at the beginning or end after the `replace` was applied

**Review by 27136773 — APPROVED** [review]
Looks good

File: src/web/mjs/connectors/HiperCool.mjs (hunk 1 above)
Comment (XenonR) [reply src/web/mjs/connectors/HiperCool.mjs]: Sounds like a good idea.

**Comment by XenonR** [comment]
I was lazy and used my old branch. How to resolve the conflict?

**Review by 27136773 — APPROVED** [review]
Looks good

**Comment by 27136773** [comment]
Conflicts can be resolved manually during merge (which must be performed by you), or by updating your branch with the commits from the latest upstream/master (reference repository manga-download/hakuneko)

**Pull request merged**